pandas/_version.py

```diff
@@ -4,9 +4,10 @@
 # directories (produced by setup.py build) will contain a much shorter file
 # that just contains the computed version number.
 
+from __future__ import annotations
```

Comment:
nit. can you move to adjacent to the other imports.

Reply:
Sure, moved it just now on top of the rest.